src/_articles/dart-vm/simd.md

```diff
@@ -1,15 +1,21 @@
 ---
 title: Using SIMD in Dart
 description: "Get more out of your CPU: operate on four numbers in parallel, using the Float32x4 class from the dart:typed_data library."
-date: 2013-08-02
+original-date: 2013-08-02
+date: 2018-07-26
 category: dart-vm
-obsolete: true
 ---
 
 _Written by John McCutchan <br>
-August 2013_
+August 2013 (note added July 2018)_
 
-Programs written in Dart can use new numeric types
+<aside class="alert alert-info" markdown="1">
+  This article's content is still relevant in Dart 2.
+  However, the examples haven't been tested recently,
+  and they use the now-optional `new` keyword.
```

Comment:
Not sure why "new" is relevant. It hasn't been removed from Dart 2.

Reply:
It hasn't been removed, but the style guide says [DON'T use `new`](https://www.dartlang.org/guides/language/effective-dart/usage#dont-use-new), and we've removed `new` from all of our auto-included code.